bears/java/JavaPMDBear.py

```diff
@@ -50,6 +51,8 @@ def create_arguments(filename, file, config_file,
                          allow_unnecessary_code: bool = False,
                          allow_unused_code: bool = False):
         """
+        :param rule_set_file:
+            Specify custom rule set file.
```

Comment:
"Custom rule set file path" or "Custom rule set file contents" would be more informative imo :)